cmd/gravity/commands/ledger.go

```diff
@@ -61,11 +61,13 @@ const (
 
 	DevNetId ChainId = "gravity-devnet"
 	CustomId ChainId = "gravity-custom"
+	MainnetId ChainId = "gravity-mainnet"
 
 	DefaultBootstrapUrl   = "http://104.248.255.124:26657"
 	DefaultPrivateRpcHost = "127.0.0.1:2500"
 
 	DefaultPersistentPeers = "2a0d75cc7833ad4780a1035b633c5bf4ef94ea4c@104.248.255.124:26656,32a091dfea2b4191d710d2609ca21a8abfe585ac@164.90.184.213:26656,34f38d98e78ed7965a56399998d9c1dccba24fe1@164.90.185.82:26656,c22e04514ce4ae0feb3480d03593d34e4713c86d@161.35.207.224:26656"
+	DefaultMainnetPersistenPeers = "b1f75f16f909b2cfc5dc108279b8c024a55a8754@134.122.37.128:26656,20310256944a853084dc26922f57c67ce3e5ee6e@45.33.3.145:26656,a071dd860f21afef888c7585d0f0185a1072bd28@212.111.41.159:26656"
```

Comment:
last letter "t" is missing, DefaultMainnetPersisten**t**Peers

Reply:
Thank you, I'll fix it!